core/controllers/topics_and_skills_dashboard.py

```diff
@@ -73,33 +73,17 @@ def get(self):
                             self.user, topic_rights)
                     )
 
-        skill_ids_for_private_skills_by_user = [
-            skill_rights.id for skill_rights in (
-                skill_services.get_unpublished_skill_rights_by_creator(
-                    self.user_id))]
-
-        skill_ids_for_unpublished_skills = [
-            skill_rights.id for skill_rights in (
-                skill_services.get_all_unpublished_skill_rights())]
-
         untriaged_skill_summary_dicts = []
         mergeable_skill_summary_dicts = []
         for skill_summary_dict in skill_summary_dicts:
             skill_id = skill_summary_dict['id']
             if (skill_id not in skill_ids_assigned_to_some_topic) and (
-                    skill_id not in skill_ids_for_unpublished_skills) and (
                         skill_id not in merged_skill_ids):
```

Comment:
Deindent by 4. Ditto for (the new) line 84.

Reply:
Done